src/documents/NodeAPI.md

```diff
@@ -53,31 +43,31 @@ Infusion includes a few small utilities to ease the process of working with a no
 
 * `name {String}` The name of your module. This should agree with its name in the npm registry.
 * `baseDir {String}` The base directory of your module. This should be the value of `__dirname` in its root directory.
 * `moduleRequire {Require}` This should be the value of `require` handed to you by node's own module loader.
 
 ### fluid.module.resolvePath(path)
 
-Resolve a path expression which may begin with a module reference of the form `${module-name}` into an absolute path relative to that module. Note that more
-modules are resolvable here than were necessarily registered with `fluid.module.register` - on startup, Infusion's node module will "pre-inspect" its filesystem path
-to the root in order to discover anything which plausibly looks like a module root - that is, it will recognise a `package.json` file which need not necessarily have a grandparent path of `node_nodules`. 
+Resolve a path expression which may begin with a module reference of the form `%module-name` into an absolute path. Note that more
+modules are resolvable here than were necessarily registered with `fluid.module.register` — on startup, Infusion's node module will "pre-inspect" its filesystem path
+to the root in order to discover anything which plausibly looks like a module root — that is, it will recognise a `package.json` file which need not necessarily have a grandparent path of `node_nodules`. 
 If the supplied path does not begin with such a module reference, it is returned unchanged.
 
-* `path {String}` A path expression to be resolved, perhaps containing symbolic module references such as `${module-name}`
+* `path {String}` A path expression to be resolved, perhaps containing symbolic module references such as `%module-name`
 * Returns: `{String}` The path expression with any symbolic module references resolved against the `fluid.module.modules` database
 
 ### fluid.require(moduleName[, foreignRequire, namespace]) 
 
 Issues node's `require` against a possibly symbolic path, and possibly also install the result at a particular global namespace path from the point of view of Infusion.
 
-* `moduleName {String}` A string be supplied to node's `require`, possibly starting with a module reference of the form `module-name` to indicate a base reference into an already
+* `moduleName {String}` A string to be supplied to node's `require`, possibly starting with a module reference of the form `%module-name` to indicate a base reference into an already
  loaded module that was previously registered using `fluid.module.register`.
 * `foreignRequire {Require}` The instance of `require` to be operated after the module name has been interpolated. If omitted, defaults to Infusion's own `require` (which may not be able to
-see everything you can - since it will very likely be higher up in the module tree than you are)
+see everything you can — since it will very likely be lower down in the module tree than you are)
```

Comment:
I'm wondering if a more general "since it will very likely be in a different location in the module tree than you are" might be better than "higher up" or "lower down"?

Otherwise, the changes all look good to me.